Pipfile

```diff
@@ -9,10 +9,10 @@ python_version = "3.5"
 [packages]
 SQLALchemy = "*"
 alembic = "*"
-securedrop-sdk = {git = "https://github.com/freedomofpress/securedrop-sdk.git"}
+securedrop-sdk = "*"
```

Comment:
ah right, so the workflow from now when we make a change on the sdk side that we need in the journalist GUI will be us making a new release of the `securedrop-sdk` and updating on PyPI, i'll do this since we need a release after this change was made: https://github.com/freedomofpress/securedrop-sdk/pull/21

Reply:
@redshiftzero Yup, I did a `0.0.1` release so that we can test this packaging story. I think we should  do a `0.0.2` release as that will allow us to experiment more as far as the version number goes.

Reply:
yep i agree